src/libraries/System.Private.CoreLib/src/System/MemoryExtensions.cs

```diff
@@ -1478,18 +1518,8 @@ private static unsafe int IndexOfAnyProbabilistic(ref char searchSpace, int sear
         /// </summary>
         /// <param name="span">The span to search.</param>
         /// <param name="values">The set of values to search for.</param>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
@Rob-Hague @adamsitnik 
Was this intentional?

[For better or worse](https://github.com/dotnet/runtime/issues/76843), the `LastIndexOfAny(ROS)` overload is marked with `AggressiveInlining`,  so this `Span` overload will now be more expensive for const values.

Reply:
It was intentional in that it became consistent with many of the other Span to ROS forwards though I agree it is (now) inconsistent with IndexOfAny(Span, ROS). I probably overlooked that.

I also believed that the annotation was superfluous in such cases - I apologise for the assumption as I did not instrument that.